description: fix build error when building against system libdrm

../../media/gpu/chromeos/video_decoder_pipeline.cc:31:10: fatal error: 'third_party/libdrm/src/include/drm/drm_fourcc.h' file not found
#include "third_party/libdrm/src/include/drm/drm_fourcc.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated


commit 5b287e0be8d8d8475ec6ef81b16eaa61e19db078
Author: Steinar H. Gunderson <sesse@chromium.org>
Date:   Tue Jan 25 19:14:53 2022 +0000

    Fix build break with system libdrm.
    
    This patch is submitted on behalf of Andres Salomon
    <dilinger@queued.net> (on Cc). Also adding him to src/AUTHORS
    per policy, as he has signed the CLA.
    
    Original patch:
    https://salsa.debian.org/dilinger/chromium/-/commit/2333c2fb985ac1ef3d7331776a7833d965c63302
    
    Change-Id: I18bd60e2e80f2b9181e74ee7972c3f457ab33b10
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3401704
    Reviewed-by: Andres Calderon Jaramillo <andrescj@chromium.org>
    Commit-Queue: Steinar H Gunderson <sesse@chromium.org>
    Auto-Submit: Steinar H Gunderson <sesse@chromium.org>
    Cr-Commit-Position: refs/heads/main@{#963119}

diff --git a/media/gpu/chromeos/BUILD.gn b/media/gpu/chromeos/BUILD.gn
index 7cb4c3ea39a16..6fe21c15ecc1b 100644
--- a/media/gpu/chromeos/BUILD.gn
+++ b/media/gpu/chromeos/BUILD.gn
@@ -25,6 +25,7 @@ source_set("chromeos") {
 
   deps = [
     "//base",
+    "//build/config/linux/libdrm",
     "//media",
     "//media/gpu:buildflags",
     "//media/gpu:command_buffer_helper",
diff --git a/media/gpu/chromeos/video_decoder_pipeline.cc b/media/gpu/chromeos/video_decoder_pipeline.cc
index d2f154ea0d87c..416b9434a8d70 100644
--- a/media/gpu/chromeos/video_decoder_pipeline.cc
+++ b/media/gpu/chromeos/video_decoder_pipeline.cc
@@ -28,8 +28,8 @@
 #include "third_party/abseil-cpp/absl/types/optional.h"
 
 #if BUILDFLAG(USE_VAAPI)
+#include <drm_fourcc.h>
 #include "media/gpu/vaapi/vaapi_video_decoder.h"
-#include "third_party/libdrm/src/include/drm/drm_fourcc.h"
 #elif BUILDFLAG(USE_V4L2_CODEC)
 #include "media/gpu/v4l2/v4l2_video_decoder.h"
 #else
